### PR TITLE
Add payment confirmation email to view details

### PR DESCRIPTION
**Pull request by Cruikshanks**
https://eaflood.atlassian.net/browse/RUBY-1159

Now we are collecting a payment confirmation email during the registration and renewal journey we should really be showing it in the view details page. Else there will be no means to see this in the system.

<details><summary>Example</summary>

Screenshot 2020-07-16 at 22 32 42

</details>

### CODE REVIEW

File: app/views/registrations/show.html.erb [hunk 1]
```diff
@@ -50,6 +50,10 @@
         <% if @registration.main_people.any? %>
           <%= render "shared/registrations/main_people_details", resource: @registration %>
         <% end %>
+
+        <% if @registration.receipt_email.present? && !@registration.receipt_email.empty? %>
```
Comment (irisfaraway) [inline app/views/registrations/show.html.erb:54]:
```suggestion
        <% if @registration.receipt_email.present? %>
```

`.present?` is more thorough so should cover us!

**Comment by Cruikshanks** [comment]
Taking it back to draft. Need to make a quick tweak.

**Comment by irisfaraway** [comment]
Oops, sorry, just saw you returned it to draft!

File: app/views/registrations/show.html.erb (hunk 1 above)
Comment (Cruikshanks) [reply app/views/registrations/show.html.erb:54]: Awesome. Had it in my head it only covered the field being there 🤦

**Comment by Cruikshanks** [comment]
SonarCloud is complaining about the accessibility elements of the table I've now added. It is right, but then we are using the table for layout and not for displaying data.

WCAG advises you don't do this and use CSS instead. However, if you do have to do it they say not to include `caption` and `scope` elements.

[F46: Failure of Success Criterion 1.3.1 due to using th elements, caption elements, or non-empty summary attributes in layout tables](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F46)

So I think we're just going to have to ignore SonarCloud on this one and ideally look to refactor to use CSS in the future (when we get those magical 5 minutes!)

**Pull request merged**